sharding-scaling/src/resources/Architecture_zh.md

```diff
@@ -12,6 +12,8 @@
 
 增量方案：
 
+- MySQL：伪装成从机，读取解析源端binlog日志来进行增量数据同步。
```

Comment:
```
Admin Guide_zh.md 
Architecture_zh.md	 
Quick Start_zh.md
```

There are only Chinese documents,  English version is need ?

Reply:
Of course, we plan add English version in #4322 .